modules/API/external/external_api.js

```diff
@@ -565,6 +565,11 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
 
                 // Since this is internal event we don't need to emit it to the consumer of the API.
                 return true;
+            case 'video-mute-status-changed':
+                if (this._onStageParticipant && this._myUserID === this._onStageParticipant) {
```

Comment:
first part of the condition seems redundant, isn't `_myUserID` always truthy in this case? Also, did you manage to test that this actually fixes the issue?

Reply:
Both are undefined on init. I tested this locally.